### PR TITLE
RFC: TaskMeter: always show number of threads

### PR DESCRIPTION
**Pull request by cgzones**
Always show the number of kernel and userland threads, even when they
are disabled to not be shown in the process list.

The data is already available and might improve understanding the system
utilization.

Use a shadow color in case the kind of thread is hidden, else the normal
meter one.

### CODE REVIEW

File: TasksMeter.c [hunk 1]
```diff
@@ -43,28 +39,20 @@ static void TasksMeter_display(const Object* cast, RichString* out) {
...
-      RichString_appendAscii(out, threadValueColor, buffer);
-      RichString_appendAscii(out, threadCaptionColor, " kthr");
-   }
+
+   RichString_appendAscii(out, settings->hideUserlandThreads ? CRT_colors[METER_SHADOW] : CRT_colors[METER_TEXT], ", ");
+   xSnprintf(buffer, sizeof(buffer), "%d", (int)this->values[1]);
+   RichString_appendAscii(out, settings->hideUserlandThreads ? CRT_colors[METER_SHADOW] : CRT_colors[TASKS_RUNNING], buffer);
+   RichString_appendAscii(out, settings->hideUserlandThreads ? CRT_colors[METER_SHADOW] : CRT_colors[METER_TEXT], " thr");
+
+
```
Comment (BenBE) [inline TasksMeter.c:50]: No need for two blank lines.

**Review by BenBE — APPROVED** [review]
LGTM.

**Review by natoscott — APPROVED** [review]
LGTM

File: TasksMeter.c (hunk 1 above)
Comment (cgzones) [reply TasksMeter.c]: dropped

**Pull request merged**